sssoja/tv-bland
# So/hide star rating

### What changes have you made?
- Fixed a bug in the `Rating` component whereby the number of full stars was always showing as half of its number rating

### Which issue(s) does this PR fix?
Fixes #63 

### Screenshots (if there are design changes)
No design changes

### How to test
Go to the show page and check:
- that the number of stars is reflective of its number rating 
- that the number of full stars is rounded up or down based on its number rating 

For example: 

<img width="593" alt="Screenshot 2020-12-06 at 22 42 11" src="https://user-images.githubusercontent.com/53219789/101293858-4ef34800-3814-11eb-8043-f4f76c62713b.png">
